Library/Homebrew/extend/pathname.rb

```diff
@@ -529,6 +529,12 @@ def unlink
     ObserverPathnameExtension.n += 1
   end
 
+  def mkpath
+    return if exist?
+    super
+    puts "mkdir #{self}" if ObserverPathnameExtension.verbose?
```

Comment:
I've changed this to use `ObserverPathnameExtension.verbose?` instead of `ARGV.verbose?`.